examples/image-builder-v6.yaml

```diff
@@ -0,0 +1,120 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# Deploying the various groups of this blueprint requires passing the output of
```

Comment:
This is written as if it was the old style where you had to manually call several commands to do multigroup. Now it all happens automatically., 

I would just say, "see instructions at <link>" and don't mention passing variables.

Reply:
Updated.